src/Tizen.NUI/src/public/Common/WeakEvent.cs

```diff
@@ -86,14 +86,15 @@ public void Invoke(object sender, object args)
         /// <summary>
         /// Invoked when the event handler count is increased.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 12 </since_tizen>
         protected virtual void OnCountIncreased()
         {
         }
 
         /// <summary>
         /// Invoked when the event handler count is decreased.
         /// </summary>
+        /// <since_tizen> 12 </since_tizen>
         protected virtual void OnCountDicreased()
```

Comment:
typo: OnCountDecreased

Reply:
Thank you for review.
I have changed as your comment.